IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs

```diff
@@ -37,23 +40,25 @@ public InfoViewModel()
       if (Preferences.ContainsKey("playMusic"))
       {
         IsPlaying = Preferences.Get("playMusic", false);
+        PauseImage = "volume_up_24px.png";
       }
       else
         IsPlaying = false;
-
+        PauseImage = "volume_off_24px.png";
     }
 
     private  async void PlayCommandExcute(object obj)
     {
-      var playMusic = obj as Switch;
-      if (playMusic.IsToggled)
+      if (!IsPlaying)
       {
         Preferences.Set("playMusic", true);
```

Comment:
Instead of using preferences, use a property inside the 'AudioService' that I mentioned above

Reply:
ok